housekeeping-vacuum-tool/src/main/java/com/hotels/housekeeping/tool/vacuum/VacuumToolApplication.java

```diff
@@ -124,6 +133,17 @@ public void run(ApplicationArguments args) {
     }
   }
 
+  private void validate(IMetaStoreClient metastore, List<Table> tables) {
+    ValidationResult validateResult = tablesValidator.validate(metastore, tables);
+    if (!validateResult.isValid()) {
+      for (ValidationFailure validationFailure : validateResult.getValidationFailures()) {
+        LOG.error("Table: " + validationFailure.getQualifiedTableName() + ". " + validationFailure.getMessage());
+        throw new RuntimeException(
+            "Configuration contains tables that are not valid for Vacuuming, please remove those tables or change validation rules");
```

Comment:
I don't want to tie in CT more, we can use this in other applications. That I have to use a CT property is just because there it is our most common use case and it will help with that.

I do notice that I throw the exception inside the loop that's wrong. (I wrote a bad test I guess)

Reply:
Oh. Makes sense in that case but can we atleast put the link to the README here so that a first time user can make some sense of the error message.